java-logging-httpcomponents/build.gradle

```diff
@@ -8,10 +8,20 @@ publishing {
     HttpComponents(MavenPublication) {
       from components.java
       groupId project.group
-      artifactId 'logging-httpcomponents'
       version project.version
     }
   }
+
+  repositories {
+    maven {
+      name = "Github"
+      url = uri("https://maven.pkg.github.com/quickcase/spring-logging")
+      credentials {
+        username = System.getenv("GH_USERNAME")
+        password = System.getenv("GH_TOKEN")
+      }
+    }
+  }
```

Comment:
To avoid duplication, this could be placed in the root `build.gradle` under `allprojects` so that you only need this code once to publish all 3 packages

Reply:
👍 moved publishing to `allprojects`